compiler/src/main/java/com/dylibso/chicory/compiler/internal/WasmAnalyzer.java

```diff
@@ -110,6 +124,43 @@ public List<CompilerInstruction> analyze(int funcId) {
                     stack.enterScope(ins.scope(), blockType(ins));
                     break;
                 case END:
+                    // Check if this is the end of a TRY_TABLE block
+                    if (ins.scope().opcode() == OpCode.TRY_TABLE) {
+                        Instruction tryTableIns = ins.scope();
+                        var operands = tryTableOperands.remove(tryTableIns.address());
+                        // Werid: sometimes we see END occur multiple times for the same try table.
```

Comment:
@andreaTP do you know why this happens?

Reply:
Shouldn't, but I'd need to check the flow, is it happening on the "happy path" e.g. no exceptions thrown?